Mathlib/Analysis/Complex/Convex.lean

```diff
@@ -34,3 +34,10 @@ lemma starConvex_ofReal_slitPlane {x : ℝ} (hx : 0 < x) : StarConvex ℝ ↑x s
 
 /-- The slit plane is star-shaped at `1`. -/
 lemma starConvex_one_slitPlane : StarConvex ℝ 1 slitPlane := starConvex_slitPlane one_pos
+
+/-- The slit plane is star-shaped at `1`, spelled out for convenience. -/
+lemma starConvex_one_slitPlane' {z : ℂ} (hz : 1 + z ∈ slitPlane) {t : ℝ} (ht : t ∈ Set.Icc 0 1) :
+    1 + t * z ∈ slitPlane := by
```

Comment:
You can use [add_smul_mem](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/Convex/Star.html#StarConvex.add_smul_mem) instead of adding a custom lemma.

Reply:
OK.